local/include/etc/confluent/docker/configure

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
```

Comment:
is this file any different than the version in cp-kafka?

Reply:
Yes. This file is making use of the golang utilities `ub` for multiple checks.
Also the mechanism of creating properties file is also different here.